app/views/admin/users/_activity.html.erb

```diff
@@ -15,7 +15,11 @@
     <p>(view full history in notes below)</p>
     <ul>
       <% @user.roles.each do |role| %>
-        <li><%= role.name %> <em><%= role.resource_name.to_s %></em></li>
+        <% if role.name === "banned" %>
```

Comment:
**suggestion (blocking):** I think a bit of JavaScript snuck in here:

```suggestion
        <% if role.name == "banned" %>
```

While they theoretically both work, `===` in Ruby is used for [case equality](https://blog.arkency.com/the-equals-equals-equals-case-equality-operator-in-ruby/), which is not quite what we want here.

Reply:
Thank you so much 🤦‍♀️